restorer.go

```diff
@@ -101,12 +96,20 @@ func (r *Restorer) Restore(cache Cache) error {
 		}
 	}
 
-	if err := g.Wait(); err != nil {
-		return errors.Wrap(err, "restoring data")
+	if api.MustParse(r.Platform.API()).AtLeast("0.8") {
+		g.Go(func() error {
+			if cacheMeta.BOM.SHA == "" {
+				return nil
+			}
+			if err := r.SBOMRestorer.RestoreFromCache(cache, cacheMeta.BOM.SHA); err != nil {
+				return err
+			}
+			return r.SBOMRestorer.RestoreToBuildpackLayers(r.Buildpacks)
```

Comment:
The benefit of introducing an abstraction is that it can _abstract away_ the orchestration logic (_arguably, even the platform API logic_).